src/api/item.ts

```diff
@@ -104,6 +107,14 @@ export const postItem = async (
         description,
         extra,
         geolocation,
+        // Define allowed settings in payload
```

Comment:
Does this mean you plan to have a setting UI at item creation to choose the placement ?

Reply:
It is already the case in the builder for the folder. But it could be good to have the same for the uploaded files. My comment here indicates that only the settings that we have allowed will be send to the backend. It is a filter to avoid to send unwanted settings to the backend, but maybe I should just send settings and let the backend choose what to do ?

```
...(settings ? {settings} : {})
```